exchanges/okgroup/okgroup_websocket.go

```diff
@@ -177,7 +205,8 @@ func (o *OKGroup) WsConnect() error {
 	return nil
 }
 
-// wsPingHandler sends a message "ping" every 27 to maintain the connection to the websocket
+// wsPingHandler sends a message "ping" every 27 to maintain the connection to
+// the websocket
 func (o *OKGroup) wsPingHandler(wg *sync.WaitGroup) {
```

Comment:
Based on their [docs](https://www.okex.com/docs/en/#ws_swap-README) this doesn't look needed anymore could remove it to save on the spawning of a go rountine